extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -273,6 +290,81 @@ export class AmpAnalytics extends AMP.BaseElement {
       return val + argList;
     });
 
+    if (request.startsWith("iframe")) {
```

Comment:
`#startsWith` is a very new ES6 feature. Might be better to do `#indexOf == 0` or `/^iframe/.test(request)`.